cmd/cli/util.go

```diff
@@ -61,3 +68,28 @@ func annotateErrorMessageWithActionableMessage(actionableMessage string, errMsgF
 
 	return errors.Errorf(errMsgFormat+actionableMessage, args...)
 }
+
+// helper function for tests that adds deployment to the clientset
+func addDeployment(fakeClientSet kubernetes.Interface, depName string, meshName string, namespace string, osmVersion string, isMesh bool) (*v1.Deployment, error) {
```

Comment:
Since this is only used by test code, kindly move it to a `_test.go` file so the actual binary doesn't use it.

Reply:
Addressed in latest changes.